gradio/examples.py

```diff
@@ -25,6 +28,27 @@
 set_documentation_group("component-helpers")
 
 
+def create_examples(
+    examples: List[Any] | List[List[Any]] | str,
+    inputs: Component | List[Component],
+    outputs: Optional[Component | List[Component]] = None,
+    fn: Optional[Callable] = None,
+    cache_examples: bool = False,
+    examples_per_page: int = 10,
+):
+    """Top-level synchronous function that creates Examples. Provided for backwards compatibility, i.e. so that gr.Examples(...) can be used to create the Examples component."""
+    examples_obj = Examples(
+        examples=examples,
+        inputs=inputs,
+        outputs=outputs,
+        fn=fn,
+        cache_examples=cache_examples,
+        examples_per_page=examples_per_page,
+    )
+    utils.synchronize_async(examples_obj.create)
```

Comment:
If this does not need to be sync, we could also run it in the background with 

            utils.run_coro_in_background(fnc, parameters)

Reply:
Actually, let's use `run_coro_in_background`

Reply:
What if the examples take a while to cache. I think this would lead to the Interface launching without the examples caching first?

Reply:
Yea agreed we should run this synchronously - like we do now

Reply:
Oh shoot, I was actually going to say 

Actually, let's use synchronize_async :D